### PR TITLE
Working eslint

### PR DESCRIPTION
**Pull request by TyHil**

### CODE REVIEW

File: src/components/Navbar.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import clsx from 'clsx';
-import Image from 'next/image';
+import Image, { StaticImageData } from 'next/image';
```
Comment (nl32) [inline src/components/Navbar.tsx:2]: This should be a type import

File: src/components/Projects.tsx [hunk 2]
```diff
@@ -121,7 +120,7 @@ export default function Projects() {
             ))}
           </div>
           <div className="pt-6">
-            <Carousel data={PROJECTS_INFO} keyBase={carouselKeyBase} ref={carouselRef}>
+            <Carousel<Project[]> data={PROJECTS_INFO} keyBase={carouselKeyBase} ref={carouselRef}>
```
Comment (nl32) [inline src/components/Projects.tsx:123]: cursed

**Review by nl32 — CHANGES_REQUESTED** [review]
The import should probably only import the type but overall changes are good

File: src/components/Projects.tsx (hunk 2 above)
Comment (TyHil) [reply src/components/Projects.tsx:123]: ikr

**Review by nl32 — APPROVED** [review]
l g t m

**Pull request merged**